routes/add-event.js

```diff
@@ -0,0 +1,19 @@
+const request = require('request');
+module.exports = (req, res) => {
+  var url = 'https://nazareth-open-tourism-platform.herokuapp.com/events';
+
+  request.post(url, (error, result) => {
```

Comment:
you do not want to be doing `request.post` here. It is the handler for the add-event form page. for the `post` request, you should create another route.